libraries/AP_OSD/AP_OSD.h

```diff
@@ -226,6 +237,15 @@ class AP_OSD_Screen : public AP_OSD_AbstractScreen
 #endif
     AP_OSD_Setting sidebars{false, 4, 5};
 
+#if AP_OSD_CRSF_PANELS_ENABLED
+    // CRSF link stats data panels
+    AP_OSD_Setting crsf_tx_power{false, 0, 0};
```

Comment:
Sorry, one more - please can you provide appropriate default positions here so that if someone enables them they show up in a sensible place in the panel.

Reply:
No problem! Done! :-)